src/Viewer.tsx

```diff
@@ -204,7 +204,7 @@ function Viewer(): ReactElement {
       ) {
         return;
       }
-      updateUrlParams(state);
```

Comment:
Previously, there was a bug where `makeDebouncedCallback` would require an argument even if the function took no arguments. I fixed the bug in `store_utils.ts` by updating the types.